src/docs/release-notes/R17/R17.mdx

```diff
@@ -0,0 +1,24 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Re-organized and added new content to explain the patterns
+-   Added more images and design specifications.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
+-   Added documentation around accessibility standards and how to design websites and illustrations.
+
+### Component Libraries
+
+-   Added new components to our [Angular](https://pxblue-components.github.io/angular), [React](https://pxblue-components.github.io/react), and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+    -   For Angular, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) and added a [nav-rail variant](https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-a-nav-rail) to our Drawer component.
+    -   For React, we now have components for [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--readme).
+    -   For React Native, we now components for [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag).
+-   Enhanced our component libraries to support right-to-left languages and developed/integrated a [RTL toggle](https://www.npmjs.com/package/@pxblue/storybook-rtl-addon) into our storybook demos.
+-   Added a new progress icon for UPS indicators and added optional labels to all progress icons.
```

Comment:
Another one I would pop under miscellaneous (we also added / will be adding a couple new lighting-related icons)

Reply:
I added a new misc section and moved the progress icons to be the first item here.